SmallLibrary/index.html

```diff
@@ -0,0 +1,66 @@
+<!--ForEach function is a void one that's why use console to see it's output-->
+
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Test JS</title>
+        <link rel='stylesheet' href="./style/body.css" type='text/css'/>
+        <link rel='stylesheet' href="./style/operation-field.css" type='text/css'/>
+    </head>
+    <body>
+        <header class="header">
+            <div class="input-form">
+                <span>Input values one bu one:</span>
+                <input class="form__text" id="array-input" type="text"/>
+                <button class="button form__button" onclick="Main.insertValue(), Main.display('.array-display__list_source', Main.getArray());">Add</button>
```

Comment:
All 'onclick' events have to be in .js files. If you use jQuery, you can use `.on` function: http://api.jquery.com/on/